### DISCUSSION TITLE
Seeing monitor add on

### DISCUSSION THREAD

**Discussion by pterodyne** [aaronwmorris/indi-allsky#604] [Ideas]
I'm writing about the idea of having an add-on feature for an additional camera pointed at the pole for a seeing monitor.  Same idea as this: (Way overpriced and windows only)

https://www.sbscientific.com/products/new-seeing-monitor/seeing-monitor-2-1/

I have built a clone of this, (QHY5-II) with 100mm Cmount lens in a security camera housing and pointed it at the pole, but without software it is pretty useless.  I think this idea has some merit, although I can't do much besides offer up my equipment for testing and provide testing.

One of the main advantages to this idea is we likely already have a Pi, Webserver, Indi, etc that would be needed.  The obvious hazard is providing enough power to the cameras.  I already have 3*pi in the observatory (2 pi zero for security cameras running ustreamer, and the main Allsky camera on a Pi4 all 3 are outside the building), so I'm trying to avoid adding more.  I also use linux/ekos/kstars for the main observatory computer on X86_64, but I think most won't, so making this as an add-on for indi-allsky makes more sense than a stand alone system.

Maybe it's just a sister project idea.  As far as I know, there are only way overpriced proprietary projects on the market (similar to allsky cameras from a few years ago!) and there isn't anything that complex that is happening.  Basically Alcor and Sm-3 do the same thing as in calculate the average seeing using very short exposures and averaging to get the real seeing.  If you search out SM-3 or alcor mini cyclops you'll see how it works.

Anyway, Id love to get this going, and I can help in any way I can, I just don't have the programming skill beyond some basic shell scripting and windows batch stuff.

Thanks for looking.  Could make this project into an all in one sky monitoring solution.

Bryan

**Comment by aaronwmorris** [comment]
I think this is an interesting idea, but I do not think it is possible to implement with a [normal] allsky camera.

Recently, I found some posts (might have been yours) related to seeing monitors on CloudyNights.  Seeing monitors use a much higher magnification than a normal allsky camera.  The SM-3 monitor uses a 100mm focal length, where an allsky camera is normally 2-3mm.  There has to be enough magnification to observe atmospheric turbulence and generate a valid FWHM measure of the stars.

I may take this on as a future project.

**Comment by pterodyne** [comment]
Thanks for replying.

I also was thinking about that idea of using the allsky camera, since we are doing long exposures, you could use the time in between to do seeing monitoring.  But I suspect the resolution would be an issue, and variable at that since some people like me have a 174mm with large pixels attached to a 185 degree lens!

having a second camera may require a powered hub with a pi, of course depending on which camera you are using for the main allsky function.

Hell lets add an inexpensive SQM sensor and sky temperature also!  :)

kidding, maybe.

I could run a cable from my seeing camera to my pi if you want to play with it ever, I can arrange access.  Right now it's plugged into my main observatory computer, but with a powered USB extension I could attach it to the allsky Pi.

**Reply by aaronwmorris** [reply]
Wow, a 174mm... I have not heard of anyone using that for an all sky camera.

My ASI290mm (2.9um pixels) with a 2.5mm lens has a resolution of ~240"/px.  That is 4 arc minutes per pixel.  Just not enough angular resolution to measure stars.

**Reply by dbainbridge** [reply]
What lens exactly are you using on that 174mm?

**Reply by pterodyne** [reply]
Fujinon FE185.  There are different ones for different sensor sizes.  I'd have to look at it to see exactly which one but it was the one that closely matched that sensor.  Even more ridiculous, at one point I had a 1600MM-Cool for a while as my allsky :) with a M43 185 fisheye.  But in the end it made more sense to sell that camera since I wasn't really using it for imaging anymore.

Ive also had an 178mm with a different lens.  And the list goes on, imaging source, Canon EOS, and even farther, way back to various stacking video cameras going back 15  or more years.  I guess I've always been obsessed with allsky cameras :)

Right now I have an old Orion Starshoot 2 Pro APS-C color that I have been wanting to get going since it's worth nothing essentially and beat to hell.  But there is no Indi driver for it that I can tell.  The driver for the sspro guider doesn't detect it.

**Reply by pterodyne** [reply]
Here is my build thread on an old unmaintained blog of mine.

https://pterodyne1.wixsite.com/website-1/post/new-all-sky-camera-design

and the lens is:

Fujinon FE185C086HA-1
